test/test_view_structures_tab.py

```diff
@@ -185,11 +185,11 @@ def test_merge_rtss(qtbot, test_object):
     # the old rtss
     roi_name = test_object.rois.get(1)["name"]
     roi_coordinates = [0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 0]
-    new_rtss = create_roi(new_rtss, roi_name, roi_coordinates, dataset)
+    new_rtss = create_roi(new_rtss, roi_name, [{'coords': roi_coordinates, 'ds': dataset}])
```

Comment:
Line too long for PEP 8 compliance, also line 192.

Reply:
Resolved. Can you please check?